examples/plot_variogram_estimation_modelling.py

```diff
@@ -33,23 +25,19 @@
 import geoutils as gu
 
 # %%
-# We start by loading example files including a difference of DEMs at Longyearbyen and the outlines to rasterize
-# a glacier mask.
-# Prior to differencing, the DEMs were aligned using :ref:`coregistration_nuthkaab` as shown in
-# the :ref:`sphx_glr_auto_examples_plot_nuth_kaab.py` example. We later refer to those elevation differences as *dh*.
+# We load example files.
 
 dh = xdem.DEM(xdem.examples.get_path("longyearbyen_ddem"))
 glacier_outlines = gu.Vector(xdem.examples.get_path("longyearbyen_glacier_outlines"))
 mask_glacier = glacier_outlines.create_mask(dh)
 
 # %%
-# We remove values on glacier terrain in order to isolate stable terrain, our proxy for elevation measurement errors.
+# We exclude values on glacier terrain in order to isolate stable terrain, our proxy for elevation errors.
 dh.data[mask_glacier] = np.nan
```

Comment:
Would it work with the `set_mask` function? If so, it would be better to use that one, to avoid confusion with NaNs and masks...

Reply:
Would need to use `set_mask` on the `Raster`, and then `gu.spatial_tools.get_array_and_mask()[0]` which is a bit long. Joins the discussion above on having a `get_array` Class method. I leave like this for now, maybe we want to update everywhere if we modify GeoUtils to shorten this.